ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -83,177 +231,128 @@ export class ImportedDocuments extends React.Component {
   };
 
   renderImportReportHeader = () => {
-    const { data, isLoading } = this.state;
+    const { data, currentFilteredRecords } = this.state;
     return (
-      <>
-        <Button
-          className="default-margin-top"
-          labelPosition="left"
-          icon="left arrow"
-          content="Import other files"
-          loading={!data}
-          disabled={!data}
-          as={Link}
-          to={CdsBackOfficeRoutes.importerCreate}
-        />
-        {this.renderCancelButton()}
-        <Divider />
-        {!data ? (
-          <>
-            <Icon loading name="circle notch" />
-            Fetching status...
-          </>
-        ) : isLoading ? (
-          <>
-            <Icon name="circle notch" loading aria-label="Import in progress" />
-            Importing literature... This may take a while. You may leave the
-            page, the process will continue in background.
-          </>
-        ) : data.status === 'SUCCEEDED' ? (
-          <>
-            <Icon name="check circle" color="green" aria-label="Completed" />
-            Literature imported successfully.
-          </>
-        ) : data.status === 'CANCELLED' ? (
-          <>
-            <Icon name="times circle" color="yellow" aria-label="Cancelled" />
-            Literature import was cancelled by the user.
-          </>
-        ) : (
-          <>
-            <Icon name="exclamation circle" color="red" aria-label="Failed" />
-            Literature import failed.
-          </>
-        )}
-        <Divider />
-      </>
+      <Grid>
+        <Grid.Column floated="left" width={12}>
+          <Button
+            className="default-margin-top"
+            labelPosition="left"
+            icon="left arrow"
+            content="Import other files"
+            loading={!data}
```

Comment:
I would use `isLoading` here and the next one, or use the `data.status`. Please remove `data` is not needed.